tests/phpunit/tests/option/wpAutoloadValuesToAutoload.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Tests for the wp_autoload_values_to_autoload function.
+ *
+ * @group Option
+ *
+ * @covers ::wp_autoload_values_to_autoload
+ */
+class Tests_Option_wpAutoloadValuesToAutoload extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 42441
+	 */
+	public function test_wp_autoload_values_to_autoload() {
+		$this->assertSameSets( array( 'yes', 'on', 'auto-on', 'auto' ), wp_autoload_values_to_autoload() );
+	}
+
+	/**
+	 * @ticket 42441
+	 */
+	public function test_wp_autoload_values_to_autoload_filter_remove() {
+
+		add_filter( 'wp_autoload_values_to_autoload', static function() {
+
+			return array( 'yes' );
+		});
```

Comment:
See failing WPCS workflow:
```suggestion
		add_filter(
			'wp_autoload_values_to_autoload',
			static function () {
				return array( 'yes' );
			}
		);
```